1.9/usage/tutorials/dcos-101/loadbalancing.md

```diff
@@ -0,0 +1,37 @@
+---
+post_title: Load-balancing
+nav_title: Load-balancing
+menu_order: 7
+---
+
+# Prerequisites
+* A [running DC/OS cluster](/docs/1.9/usage/tutorials/dcos-101/cli/) with [the DC/OS CLI installed](link to installation instructions].
+* [app2 and Marathon-LB](/docs/1.9/usage/tutorials/dcos-101/app2/) deployed and running in your cluster.
+
+# Objective
+In this final session, we will scale our application to multiple instances and learn how internal and external services choose which instance to use once the application has been scaled.
+
+# Steps
+Load-balancers decide which instance of an app internal or external services should use. With DC/OS, you have two different built-in load-balancer options: : [Marathon-LB](/docs/1.9/usage/service-discovery/marathon-lb/) and [Named VIPs](/docs/1.9/usage/service-discovery/load-balancing-vips/).
+We have seen both before in the context of service discovery and especially marathon-lb when making app2 publically available.
```

Comment:
`We have already explored these load balancers in the context of service discovery, when we used Marathon-LB to publicly expose app2. Now let's explore them a bit more.`